pontoon/sync/vcs/models.py

```diff
@@ -556,6 +556,25 @@ def resource_paths_without_config(self):
                 if is_resource(filename):
                     yield os.path.join(root, filename)
 
+    @property
+    def changed_config_files(self):
+        """
+        Return True, when a project has PC enabled and the project config files change.
+        """
+        if not self.db_project.configuration_file:
+            return False
+
+        config_files = set(
+            pc.path.replace(os.path.join(self.source_directory_path, ""), "")
+            for pc in self.configuration.parsed_configuration.configs
+        )
+        changed_files = set(self.changed_source_files[0])
+
+        if changed_files & config_files:
```

Comment:
Ohhhhh, now I see what you did here: `&` causes the result to be an [intersection](https://docs.python.org/2/library/sets.html#set-objects) of the two sets!

Can you please use `a.intersection(b)` or at least add a comment? I find this syntax very confusing for beginners like myself. :-)

Reply:
I would do this to be more explicit:
```python
changed_config_files = changed_files.intersection(config_files)

if changed_config_files:
    return True

return False
```